tests/kinetic_sdk_test.py

```diff
@@ -13,7 +13,8 @@
 sdk = KineticSdk.setup(
     endpoint='http://localhost:3000',
     environment='local',
-    index=1
+    index=1,
+    headers=[{ 'Extra-Header': 4 }]
```

Comment:
The only custom headers that are sent through with the webhooks are the ones prefixed with `kinetic`, so in this example it should be `kinetic-extra-header`.

Reply:
> The only custom headers that are sent through with the webhooks are the ones prefixed with `kinetic`, so in this example it should be `kinetic-extra-header`.

thanks, I updated it.